pytket/tests/qasm_test.py

```diff
@@ -329,16 +330,6 @@ def test_output_error_modes() -> None:
         c.add_qubit(Qubit("q"))
         circuit_to_qasm_str(c)
     assert "OPENQASM registers must use a single index" in str(errorinfo.value)
-    with pytest.raises(Exception) as errorinfo:
-        c = Circuit(2)
-        c.CV(0, 1)
-        circuit_to_qasm_str(c)
-    assert "Cannot print command of type: CV" in str(errorinfo.value)
-    with pytest.raises(Exception) as errorinfo:
-        c = Circuit(2)
-        c.add_gate(OpType.ZZMax, [0, 1])
-        circuit_to_qasm_str(c, header="qelib1")
-    assert "Gate of type ZZ is not defined in header qelib1.inc" in str(errorinfo.value)
```

Comment:
I've done this now.
For your second request, `test_non_lib_gates` should already be doing what you're requesting I think, though please let me know if you think its missing something.
For the first bit, I've added back the tests to confirm that CV and ZZMax do the right thing. This was a good suggestion because the ZZMax gate has different output depending on the library, so I've now added an additional check to confirm the gate is in the header before adding.